DEBUG 2017-03-17 11:59:05,815: 	 Executing command: 'submit'
DEBUG 2017-03-17 11:59:05,823: 	 Found CRAB cache file /afs/cern.ch/user/r/rverma/.crab3
DEBUG 2017-03-17 11:59:05,861: 	 Checking credentials
DEBUG 2017-03-17 11:59:05,862: 	 Found proxy file /tmp/x509up_u93032
DEBUG 2017-03-17 11:59:05,862: 	 Getting proxy life time left
DEBUG 2017-03-17 11:59:07,628: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u93032 -timeleft
 output : 39106

 error: 
 retcode : 0
DEBUG 2017-03-17 11:59:08,560: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u93032 -actimeleft
 output : 39102

 error: 
 retcode : 0
DEBUG 2017-03-17 11:59:08,560: 	 Proxy valid for 10:51:46 hours
DEBUG 2017-03-18 10:57:03,022: 	 Executing command: 'submit'
DEBUG 2017-03-18 10:57:03,036: 	 Found CRAB cache file /afs/cern.ch/user/r/rverma/.crab3
DEBUG 2017-03-18 10:57:03,052: 	 Checking credentials
DEBUG 2017-03-18 10:57:03,053: 	 Found proxy file /tmp/x509up_u93032
DEBUG 2017-03-18 10:57:03,053: 	 Getting proxy life time left
DEBUG 2017-03-18 10:57:03,973: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u93032 -timeleft
 output : 42407

 error: 
 retcode : 0
DEBUG 2017-03-18 10:57:04,784: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u93032 -actimeleft
 output : 42406

 error: 
 retcode : 0
DEBUG 2017-03-18 10:57:04,785: 	 Proxy valid for 11:46:47 hours
DEBUG 2017-03-18 10:57:04,786: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
DEBUG 2017-03-18 11:02:45,209: 	 Executing command: 'submit'
DEBUG 2017-03-18 11:02:45,216: 	 Found CRAB cache file /afs/cern.ch/user/r/rverma/.crab3
DEBUG 2017-03-18 11:02:45,229: 	 Checking credentials
DEBUG 2017-03-18 11:02:45,232: 	 Found proxy file /tmp/x509up_u93032
DEBUG 2017-03-18 11:02:45,232: 	 Getting proxy life time left
DEBUG 2017-03-18 11:02:46,546: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u93032 -timeleft
 output : 42064

 error: 
 retcode : 0
DEBUG 2017-03-18 11:02:47,703: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u93032 -actimeleft
 output : 42063

 error: 
 retcode : 0
DEBUG 2017-03-18 11:02:47,703: 	 Proxy valid for 11:41:04 hours
DEBUG 2017-03-18 11:02:47,705: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
DEBUG 2017-03-18 12:09:38,917: 	 Executing command: 'kill'
DEBUG 2017-03-18 12:09:38,917: 	 Found CRAB cache file /afs/cern.ch/user/r/rverma/.crab3
DEBUG 2017-03-18 12:09:38,931: 	 Checking credentials
DEBUG 2017-03-18 12:09:38,931: 	 Found proxy file /tmp/x509up_u93032
DEBUG 2017-03-18 12:09:38,931: 	 Getting proxy life time left
DEBUG 2017-03-18 12:09:39,537: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u93032 -timeleft
 output : 40965

 error: 
 retcode : 0
DEBUG 2017-03-18 12:09:40,144: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u93032 -actimeleft
 output : 40964

 error: 
 retcode : 0
DEBUG 2017-03-18 12:09:40,145: 	 Proxy valid for 11:22:45 hours
ERROR 2017-03-18 12:09:40,147: 	 Cannot find .requestcache file in CRAB project directory /afs/cern.ch/work/r/rverma/private/miniAOD_to_ntuple_13TeV/CMSSW_8_0_25/src/CrabNtuple/test/CrabEleData_20170317/crab_QCD_Ntuple_EleData_20170317
ERROR 2017-03-18 12:09:40,147: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 321, in __init__
    self.loadLocalCache(proxyOptsSetPlace)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 450, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/cern.ch/work/r/rverma/private/miniAOD_to_ntuple_13TeV/CMSSW_8_0_25/src/CrabNtuple/test/CrabEleData_20170317/crab_QCD_Ntuple_EleData_20170317
DEBUG 2017-03-19 15:34:16,253: 	 Executing command: 'status'
DEBUG 2017-03-19 15:34:16,256: 	 Found CRAB cache file /afs/cern.ch/user/r/rverma/.crab3
ERROR 2017-03-19 15:34:16,259: 	 Error: CrabMuMC_20170318/crab_QCD_Ntuple_MuMC_20170318 is not a valid CRAB project directory.
ERROR 2017-03-19 15:34:16,259: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 257, in __init__
    self.validateOptions()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 762, in validateOptions
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 566, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: CrabMuMC_20170318/crab_QCD_Ntuple_MuMC_20170318 is not a valid CRAB project directory.
DEBUG 2017-03-19 15:34:39,399: 	 Executing command: 'status'
DEBUG 2017-03-19 15:34:39,399: 	 Found CRAB cache file /afs/cern.ch/user/r/rverma/.crab3
DEBUG 2017-03-19 15:34:39,414: 	 Checking credentials
DEBUG 2017-03-19 15:34:39,415: 	 Found proxy file /tmp/x509up_u93032
DEBUG 2017-03-19 15:34:39,415: 	 Getting proxy life time left
DEBUG 2017-03-19 15:34:40,641: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u93032 -timeleft
 output : 43144

 error: 
 retcode : 0
DEBUG 2017-03-19 15:34:41,671: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u93032 -actimeleft
 output : 43142

 error: 
 retcode : 0
DEBUG 2017-03-19 15:34:41,671: 	 Proxy valid for 11:59:04 hours
ERROR 2017-03-19 15:34:41,673: 	 Cannot find .requestcache file in CRAB project directory /afs/cern.ch/work/r/rverma/private/miniAOD_to_ntuple_13TeV/CMSSW_8_0_25/src/CrabNtuple/test/CrabMuMC_20170318/crab_QCD_MuMC_20170318
ERROR 2017-03-19 15:34:41,673: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 321, in __init__
    self.loadLocalCache(proxyOptsSetPlace)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1703/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 450, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/cern.ch/work/r/rverma/private/miniAOD_to_ntuple_13TeV/CMSSW_8_0_25/src/CrabNtuple/test/CrabMuMC_20170318/crab_QCD_MuMC_20170318
